akka-actor-tests/src/test/scala/akka/pattern/StatusReplySpec.scala

```diff
@@ -66,7 +68,7 @@ class StatusReplySpec extends AkkaSpec with ScalaFutures {
       val result = probe.ref.askWithStatus("request")
       probe.expectMsg("request")
       probe.lastSender ! StatusReply.Success("woho")
-      result.futureValue should ===("woho")
+      Await.result(result, timeout.duration) should ===("woho")
```

Comment:
Puzzled as well, don't recall by heart the problem, but I decided to go safe on `Await` since it's widely used in the test codebase

Reply:
could be worth looking into because our current preference is to use futureValue and that is much more used in new tests, such as those for Akka Typed

Reply:
This fail with:
```
[error] -- [E134] Type Mismatch Error: /Users/andreatp/workspace/tmp/akka/akka-actor-tests/src/test/scala/akka/pattern/StatusReplySpec.scala:71:13 
[error] 71 |      result.futureValue shouldBe "woho"
[error]    |      ^^^^^^^^^^^^^^^^^^
[error]    |None of the overloaded alternatives of method futureValue in trait FutureConcept with types
[error]    | (implicit config: StatusReplySpec.this.PatienceConfig, pos: 
[error]    |  org.scalactic.source.Position
[error]    |): Any
[error]    | (interval: org.scalatest.concurrent.PatienceConfiguration.Interval)
[error]    |  (implicit config: StatusReplySpec.this.PatienceConfig, pos: 
[error]    |    org.scalactic.source.Position
[error]    |  ): Any
[error]    | (timeout: org.scalatest.concurrent.PatienceConfiguration.Timeout)
[error]    |  (implicit config: StatusReplySpec.this.PatienceConfig, pos: 
[error]    |    org.scalactic.source.Position
[error]    |  ): Any
[error]    | (timeout: org.scalatest.concurrent.PatienceConfiguration.Timeout, interval: 
[error]    |  org.scalatest.concurrent.PatienceConfiguration.Interval
[error]    |)(implicit pos: org.scalactic.source.Position): Any
[error]    |match expected type ?{ shouldBe: ? }
[error] one error found
```

Those are very likely rough edges of ScalaTest on Scala 3, I will follow up with a ticket there.